src/one_dragon/base/push/channel/chronocat.py

```
@@ -0,0 +1,228 @@
+
+        Args:
+            url: API 地址
+            headers: 请求头
+            data: 请求数据
+
+        Returns:
+            bool: 是否发送成功
+        """
+        try:
+            response = requests.post(url, headers=headers, data=json.dumps(data), timeout=15)
+            return response.status_code == 200
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)